table/queries/merged_reduced_scans.sql

```diff
@@ -64,8 +72,8 @@ SELECT
     Grouped.* EXCEPT (country_code),
     IFNULL(country_name, country_code) AS country_name,
     CASE
-        WHEN (STARTS_WITH(outcome, "expected/")) THEN 0
-        WHEN (STARTS_WITH(outcome, "dial/") OR STARTS_WITH(outcome, "setup/") OR ENDS_WITH(outcome, "/invalid")) THEN NULL
+        WHEN (STARTS_WITH(outcome, "✅")) THEN 0
+        WHEN (STARTS_WITH(outcome, "❗️dial/") OR STARTS_WITH(outcome, "setup/") OR ENDS_WITH(outcome, "/invalid")) THEN NULL
         WHEN (ENDS_WITH(outcome, "unknown")) THEN count / 2.0
         ELSE count
```

Comment:
Let's use ❗️ for unexpected results only, so it's clearer. It would be nice to see something like this, with outcome aligned with expected evaluation:

```suggestion
        WHEN (STARTS_WITH(outcome, "✅")) THEN 0
        WHEN (STARTS_WITH(outcome, "❗️")) THEN count
        WHEN (STARTS_WITH(outcome, "❔")) THEN NULL
        ELSE count / 2.0  # unknown
```